scrunch/datasets.py

```diff
@@ -173,6 +173,15 @@ def get_dataset(dataset, connection=None, editor=False, project=None):
         ds.change_editor(authenticated_url)
     return ds
 
+def get_project_by_id(project_id, connection=None):
```

Comment:
This can be a class method of the `Project` class. It is very suitable to be in the Project instead of dealing with it as a stand alone function, I think that is a misdesign of the original Scrunch.

```
project = Project.get_by_id()


...


class Project:
   ...
    @classmethod
   def get_by_id(cls, project_id):
         ....
         return cls(resource)

```